buildcmake

```diff
@@ -514,6 +520,8 @@ fi
 CC=$opt_CC CXX=$opt_CXX FC=$opt_FC cmake "$my_srcdir" \
   -G "Unix Makefiles" \
   -DCMAKE_BUILD_TYPE="$opt_production" \
+  -DADDITIONAL_INCLUDE_DIRS="$opt_incdir_raw" \
+  -DADDITIONAL_LIB_DIRS="$opt_libdir_raw" \
```

Comment:
Where does this `ADDITIONAL_LIB_DIRS` variable get used?

Reply:
I was waiting for feedback regarding whether this approach is viable before using this variable